js/data/odata/utils.js

```diff
@@ -324,8 +324,8 @@ const interpretJsonFormat = (obj, textStatus, transformOptions, ajaxOptions) =>
     }
 
     const value = 'd' in obj && (Array.isArray(obj.d) || isObject(obj.d))
-        ? interpretVerboseJsonFormat(obj, textStatus)
-        : interpretLightJsonFormat(obj, textStatus);
```

Comment:
why textStatus is removed here? is behaviour the same without it?

Reply:
Yes. Both functions interpretVerboseJsonFormat() and interpretLightJsonFormat() accepts only 1 argument.